framework/resources/Functional/int96/q28.e

```diff
@@ -1 +1 @@
-.*numFiles=2, usedMetadataFile=true.*
+.*numFiles=2.*usedMetadataFile=true.*
```

Comment:
This is too relaxed. Need to verify numRowGroups as well. I already have a fix for this task. You can either change your PR or close it. I will post a PR.

Reply:
Or change yours to the following, thanks.

.*numFiles=2
.*numRowGroups=2
.*usedMetadataFile=true

Reply:
I don't know what is your process for integration, but if I change to what your propose it means the PR in drill and in the test framework has to be merged at the same time. If one of them is merged and not the other, the test will fail.

Reply:
Yes, we merge in sync.